.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,6 +3,7 @@
 
 <!-- (See our contributing guidelines for more details) -->
 ## Checklist:
+- [ ] Did you sign the [Contributor License Agreement](https://cla-assistant.io/wwt/SwiftCurrent)?
```

Comment:
Are you thinking this will encourage them to sign the CLA before submitting the PR and having the assistant tell them?

Reply:
Yup, my reasoning was basically that I as a contributor would want to look over this checklist before submitting the PR for review.